builtin/providers/aws/validators_test.go

```diff
@@ -553,3 +553,58 @@ func TestValidateDbEventSubscriptionName(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateJsonString(t *testing.T) {
+	type testCases struct {
+		Value    string
+		ErrCount int
```

Comment:
Very true, albeit I kept it like that for sake of consistency with other table-based tests I've seen in the code.